src/components/mote/components/innkalling/DialogmoteMoteStatusPanel.tsx

```diff
@@ -73,7 +89,15 @@ export const DialogmoteMoteStatusPanel = ({ dialogmote }: Props) => {
         <DeltakereSvarInfo dialogmote={dialogmote} />
       </FlexRow>
 
-      <FlexRow topPadding={PaddingSize.MD}>{texts.infoMessage}</FlexRow>
+      <FlexRow topPadding={PaddingSize.SM}>{texts.infoMessage}</FlexRow>
+
+      {noNarmesteLeder && (
+        <FlexRow topPadding={PaddingSize.SM}>
+          <AlertstripeFullbredde type="advarsel">
+            <p className="sist">{texts.noNarmesteleder}</p>
```

Comment:
Samme som over, styled-component i stedet for className, evt komponent herfra https://old-design-nav.vercel.app/designsystem/components/typografi